capitolwords_ng/parsers/crec.py

```diff
@@ -111,24 +105,28 @@ def parse_mods_file(self, mods_file):
         for record in records:
             if (record['ID'].split('-')[-1].startswith('PgD') or
                 record['ID'].split('-')[-2].startswith('PgD') or
+                record['ID'].split('-')[-1].startswith('FrontMatter') or
                 record['content'] is None or
                 'content' not in record.keys()):
-                # dont process daily digests
+                # dont process daily digests or front matters or pages with no content
```

Comment:
If we can identify the specific doc type in some way we could include that info in the logs, so it shows that this was not parsed because it wasn't the right doc type.